### PR TITLE
[17.0][ADD] Add Stock Account Show Automatic Valuation Module

### PR DESCRIPTION
**Pull request by cvinh**
The field `group_stock_accounting_automatic` has been hidden by Odoo SA and makes Automatic Valuation impossible in CE
This module makes it shown

### CODE REVIEW

**Comment by cvinh** [comment]
Hello, is there anybody to review ? this is a very simple module...
@pedrobaeza please I know you like real time valuation ;-)

File: stock_account_show_automatic_valuation/readme/DESCRIPTION.md [hunk 1]
```diff
@@ -0,0 +1 @@
+This module show the "Automatic Valuation" setup for stock that has been hidden by Odoo SA in Community Edition
```
Comment (rvalyi) [inline stock_account_show_automatic_valuation/readme/DESCRIPTION.md]: s/show/shows

**Review by rvalyi — COMMENTED** [review]
Minor English mistake. I would approve it otherwise.

**Review by pedrobaeza — APPROVED** [review]
Apart from Raphäel's comment.

**Comment by cvinh** [comment]
I rebased as well

**Review by rvalyi — APPROVED** [review]

**Pull request merged**